logs/application.log

```diff
@@ -0,0 +1 @@
+
```

Comment:
did you fetch from upstream? dont commit logs, and the gitignore should also include logs folder once you fetch.

Reply:
Will handle it in another PR, as we will have to change the JUnit tests also